build.gradle

```diff
@@ -110,33 +110,54 @@ distZip.dependsOn 'copy'
 
 
 dependencies {
+    // Spring Boot
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-jetty'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-autoconfigure'
+    implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation 'io.springfox:springfox-boot-starter:3.0.0'
+
+    // MySQL
+    implementation 'mysql:mysql-connector-java:5.1.47'
+
+    // Logger
     implementation 'org.slf4j:slf4j-api:1.7.10'
     implementation 'org.slf4j:slf4j-log4j12:1.7.10'
+
+    // Presto
     implementation 'com.facebook.presto:presto-client:0.215'
     implementation 'io.prestosql:presto-client:347'
     implementation 'io.prestosql:presto-parser:347'
     implementation 'com.squareup.okhttp3:okhttp'
+
+    // Spark
     implementation 'org.apache.httpcomponents:fluent-hc'
-    compileOnly 'org.projectlombok:lombok:1.18.8'
-    annotationProcessor 'org.projectlombok:lombok:1.18.8'
-    implementation 'org.komamitsu:fluency:1.7.0'
+    implementation 'org.jsoup:jsoup:1.11.3'
```

Comment:
No confidence about relating to Spark or not
https://search.maven.org/artifact/org.jsoup/jsoup/1.11.3/jar

Reply:
Actually, It's not part of Spark, but Yanagishima's Spark code uses it.